server/services/Registry.js

```diff
@@ -13,17 +13,57 @@ type DbResponse<R> = {|
   rowsAffected: Array<number>,
 |};
 
+//should match SQL output
 export type DogLicense = {|
-  CertificateID: number,
-  'Registered Number': string,
-  InOut: 'I' | '*' | '#',
-  'Date of Death': ?string,
-  'Decedent Name': string,
-  'Last Name': string,
-  'First Name': string,
-  RegisteredYear: string,
-  AgeOrDateOfBirth: string,
-  Pending: number,
+  CertificateID: string,
```

Comment:
The reason it's still there is that it is included in later code in Registry.js that does data output stuff, and when I remove it it throws errors that require deleting lots of code :/ thoughts?